notebooks/ch17/classification_multiclass.md

```diff
@@ -281,32 +290,40 @@ lr.fit(X_train, y_train)
+
+
+```python
+# HIDDEN
+sns.lmplot(x='sepal_length', y='sepal_width', data=iris, hue='species', fit_reg=False);
+```
 
 
-    array([[19,  0,  0],
-           [ 0, 15,  2],
-           [ 0,  0, 17]])
+![png](classification_multiclass_files/classification_multiclass_22_0.png)
 
 
+The `Iris-versicolor` and `Iris-virginica` points overlap for these two features. Though the remaining features (`petal_width` and `petal_length`) contribute additional information to help distinguish between the two classes, our classifier still misclassified the two observations.
 
-The confusion matrix shows that our classifier misclassified two `Iris-versicolour` observations as `Iris-virginica`. In the real world, certain misclassifications can be more common than others. Confusion matrices are valuable because they help us identify where our classifier struggles, and thus provides insight on what kind of features we may need to extract in order to improve the classifier.
+Likewise in the real world, misclassifications can be common if two classes bear similar features. Confusion matrices are valuable because they help us identify the errors that our classifier makes, and thus provides insight on what kind of additional features we may need to extract in order to improve the classifier.
 
 ## Multilabel Classification
 
-Another type of classification problem is **multilabel classification**, in which each observation can have multiple labels. An example would be a document classification system: a document can be classified as having positive or negative sentiments, but it can also be distinguished between religious/nonreligious or political/apolitical. Multilabel problems can also be multiclass; we may want our document classification system to distinguish between a list of genres, as well as identify the language that the document is written in. 
+Another type of classification problem is **multilabel classification**, in which each observation can have multiple labels. An example would be a document classification system: a document can have positive or negative sentiment, religious or nonreligious content, and liberal or conservative leaning. Multilabel problems can also be multiclass; we may want our document classification system to distinguish between a list of genres, or identify the language that the document is written in. 
 
-Because each set of labels in a multiclass classification is mutually exclusive, each problem can be solved independently of the others. We can then combine the outputs so that every observation is assigned a set of classification labels.
+Because each set of labels in a multilabel classification is mutually exclusive, we may perform multilabel classification by simply training a separate classifier on each set of labels. To label a new point, we combine each classifier's predictions.
```

Comment:
This is a mouthful; I'll take out the dependent clause and start the sentence with "We may perform..." in the next commit.